code/game/turfs/simulated/wall/mineral_walls.dm

```diff
@@ -139,6 +139,18 @@
 	explosion_block = 0
 	canSmoothWith = list(/turf/closed/wall/mineral/wood, /obj/structure/falsewall/wood)
 
+/turf/closed/wall/mineral/wood/try_decon(obj/item/W, mob/user, turf/T)
+	if(istype(W, /obj/item/twohanded/fireaxe))
+		var/obj/item/twohanded/fireaxe/A = W
+		if(!iswallturf(src) || !user || !W || !A)
+			return TRUE
+		if( user.loc == A && user.get_active_held_item() == W )
```

Comment:
I suspect the user's loc is never going to be their fireaxe

Reply:
There have been admin shenanigans before.